web/src/styles/base/border-colors.scss

```diff
@@ -29,6 +29,10 @@
   border-color: $secondary-light !important;
 }
 
+.border-accent-cool-light {
```

Comment:
[dust] I went through an exercise of creating elements with background color, text color, and border color for all of the colors that are defined in the design system within [storybook](http://nj-bfs-design-storybook.s3-website-us-east-1.amazonaws.com/?path=/story/atoms-colors--colors), and then manually created the css attribute names for the ones that did not exist.

I think this attribute is not needed. If it is, any idea why it's not rendering before you added it?

Reply:
I'm not sure I understand - I only added it because it didn't exist yet in this file. Are you saying the color is missing from storybook?